cl/users/models.py

```diff
@@ -284,6 +284,11 @@ class BackoffEvent(AbstractDateTimeModel):
     next_retry_date = models.DateTimeField(
         help_text="The next retry datetime for exponential backoff events.",
     )
+    event_sub_type = models.SmallIntegerField(
+        help_text="The SNS bounce subtype that triggered the object.",
```

Comment:
What does SNS stand for?

Reply:
yes, SNS stands for Amazon "Simple Notification Service", the service that sends us notifications about the messages we sent, actually, I might change it to "The SES bounce subtype..." at the end SES is responsible for creating that notification.